### DISCUSSION TITLE
Customize login screen

### DISCUSSION THREAD

**Discussion by daniel-nlm** [prasanthrangan/hyprdots#296] [Q&A]
Hi! I've just switch to Arch and Hyprland with this setup install, it so cool and easy to get use to imo. 
However I can't find anyway to change to login screen theme
How can I change the theme ( default is Catppucin-mocha ig ) 
How can I change the user's avatar as well?

Looking for help. Im still very new to Arch as well as Hyprland. btw, this set up is so cool, especially for pokemon fan like me ;)

**Comment by T-Crypt** [comment]
@daniel-nlm 

1. You can change the login wallpaper using the sddmwall.sh script in `~/.config/hypr/scripts/sddmwall.sh` which will change the login screen to the current wallpaper set (so pick your favorite wallpaper and run the script manually)
2. You can modify the current theme that is present in `/usr/share/sddm/themes/corners` 
3. You'll need to mkdir in /usr/share/sddm/ named 'faces'.

Youll need to enable avatars in 
`/etc/sddm.conf.d/kde_settings.conf`

Under block [Theme].

```
[Theme]
EnableAvatars=true
FacesDir=/usr/share/sddm/faces
``` 

Then simply copy the file you want as your user picture to in the faces diretory /usr/share/sddm/faces with the file extension of .face.icon

**Reply by daniel-nlm** [reply]
TYSM, it work very well.
I want to ask more about login screen theme. So i have to configure it all myself `/usr/share/sddm/themes/corners/theme.conf`
I don't know much about color scheme T-T

**Reply by T-Crypt** [reply]
@daniel-nlm 

Are you trying to change the colors on the login screen?

You can use the color-hex link below to get colors for the theme.conf or you can use the kde store to download and use a separate theme! NOTE: The sddmwall.sh won't work for your new theme so youll have to update the background manually using the backgrounds folder and modifying the theme.conf in the new theme.

If you decide to use a new theme, you'll need to update the `/etc/sddm.conf.d/kde_settings.conf` to include the new theme and modify as necessary.

https://www.color-hex.com/

https://store.kde.org/browse?cat=101&ord=latest
